src/pages/documentation/consent-for-edge-network/tabs/api-reference.md

```diff
@@ -16,12 +16,6 @@ String extensionVersion = Consent.extensionVersion();
 
 #### Kotlin
 
-**Syntax**
-
-```java
-fun extensionVersion(): String
-```
-
```

Comment:
Same here - I thought that these _were_ the correct Kotlin. Unless we're deciding to, en masse, remove all Kotlins regardless of their correctness, since we don't have all the Kotlin syntaxes in yet (which is also fine)